src/sync/installation.ts

```diff
@@ -370,6 +371,9 @@ export const processInstallation =
       }
 
       await subscription.update({ syncStatus: 'FAILED' });
+
+      statsd.increment('installation-sync.failed');
```

Comment:
Do we count successful syncronisations as well?

Would be great to count total sync durations (if we have the start time in our queue messages). It is nice to have. We should add it in the future